yield_plugin/src/yield_plugin.cpp

```diff
@@ -417,8 +418,20 @@ namespace yield_plugin
       double collision_time = 0; //\TODO comming from carma_wm collision detection in future (CAR 4288)
 
       double goal_velocity = rwol_collision[0].object.velocity.twist.linear.x;
-      
-      double goal_pos = x_lead - std::max(goal_velocity * gap_time, config_.x_gap); 
+      // determine the safety inter-vehicle gap based on speed
+      double safety_gap = std::max(goal_velocity * gap_time, config_.x_gap);
+      if (config_.enable_adjustable_gap)
+      {
+        // check if a digital_gap is available
+        double digital_gap = check_traj_for_digital_min_gap(original_tp);
+        // if a digital gap is available, it is replaced as safety gap 
+        if (digital_gap < std::numeric_limits<double>::max())
```

Comment:
Does it make sense to also compare to the current configured gap and use the larger of the two? Or do we want CARMA Cloud to be able to tell vehicles to shrink their headway as well? There might be benefits both ways.

Reply:
At the moment the regulatory element is strictly a minimum gap. We would need to update the implementation to support a max gap which is not currently even part of the traffic control message spec.

Reply:
@msmcconnell it already is updated to minimum gap in the new commit.

Reply:
Yeah what you have is fine Saina, I was mainly referring to Kyle's question on if carma-cloud would provide max headways (forced gap shrinkage). The answer is it will not.